client/src/st_stuff.cpp

```diff
@@ -701,7 +701,7 @@ bool ST_Responder (event_t *ev)
 			if (gamemode == retail_chex)
 				sprintf(buf,"1%c",buf[1]);
 
-            sprintf (buf + 3, "map %s\n", buf);
+            sprintf (buf + 3, "map %.7s\n", buf);
```

Comment:
I think `%.7s` is a bit misleading to the reader as they may initially think it refers to how many characters will be written to the string. Since it should always be writing 2 characters, would `%.2s` work (eg, properly terminate the string)?

Reply:
That does work, and commit sent.